packages/devtools_app/integration_test/run_tests.dart

```diff
@@ -17,14 +17,6 @@ import 'test_infra/run/run_test.dart';
 const _testDirectory = 'integration_test/test';
 const _offlineIndicator = 'integration_test/test/offline';
 
-/// The set of test that should be skipped for all devices.
-///
-/// This list should be empty most of the time, but may contain a broken test
-/// while a fix being worked on.
-const _skipTests = <String>[
-  'my_example_test.dart',
-];
-
 void main(List<String> args) async {
```

Comment:
why are we removing this skip mechanism? we should leave this here so that we can skip integration tests in the future

Reply:
reverted